libs/usb-drive/src/mock_file_usb_drive.ts

```diff
@@ -0,0 +1,18 @@
+import { UsbDrive, UsbDriveStatus } from './usb_drive';
+
+/**
+ * TODO: Implement properly to enable USB drive operations in integration tests.
+ */
+export class MockFileUsbDrive implements UsbDrive {
```

Comment:
I'm excited about this as it will allow us to revive and expand `admin` integration tests, but there's already a lot in this PR so punting on implementation anything but `no_drive`.